### DISCUSSION TITLE
Pin a number of Loras in memory?

### DISCUSSION THREAD

**Discussion by biship** [vladmandic/sdnext#882] [Q&A] [answered]
I change my prompts often (as well as removing or adding Loras) and I see automatic keeps loading them when I generate:

Loading weights: D:\Downloads\Code\automatic\models\Lora\vsTanliner_v15.safetensors ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 0.0/9.6 MB -:--:--

Is there a way to pin a configurable number of them into GPU memory?

**Comment by vladmandic** [comment, marked as answer]
Not at the moment, but that would not be a bad thing to have. I'll take a look.

**Comment by Aptronymist** [comment]
I was just having that thought today after doing some heavy lora shuffling wildcards last night to create some of the most horrifying images I've ever made, it would be nice to have at least a lora/hypernetwork/embed cache or something, but pinning them in memory would be even better. In fact, if that becomes an option, I might try out the GPU-based ramdrive I saw the other day...
